### ISSUE TITLE
If you use parentheses mark in Font Name field the font won't load.

### ISSUE THREAD

**Issue by shahabsiavash** [kontur/fontsampler-wordpress-plugin#168]
I realized it just now, I don't know. It must be new, cause suddenly 4 of my font samplers are not working, just had a hunch maybe it's because of parentheses mark, and it was it!

**Comment by kontur** [comment]
Yes, there really should be more explicit checking of the font names. Thanks for reporting 👍

**Comment by kontur** [comment]
Same as #148 in essence.
